stream-chat-android-client/build.gradle

```diff
@@ -58,6 +58,7 @@ tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
 
 dependencies {
     api project(':stream-chat-android-core')
+    debugApi project(":libraries:logging:stream-logger-android")
```

Comment:
I think we don't need to include this here as it was already added in `core` module

Reply:
This one brings an additional dependency `stream-logger-android` which contains `Logcat` wrapper, `File` wrapper and also enables the logging